### PR TITLE
docs(statsd) new metrics and flexible configurability

### PR DESCRIPTION
**Pull request by thibaultcha**

### CODE REVIEW

**Comment by thibaultcha** [comment]
https://github.com/Mashape/kong/pull/2090

File: app/plugins/statsd.md [hunk 1]
```diff
@@ -54,17 +52,40 @@ parameter                     | default | description
...
-`status_count`               | For each status code returned, increment its counter by 1 | kong.\<api_name>.\<http_status_code>.count
-`unique_users`               | count of users made a request to the api | kong.\<api_name>.user.uniques
-`request_per_user`               | For each request by the user, increment its counter by 1 | kong.\<api_name>.\<consumer_id>.count
+---                        | ---         | ---
+`request_count`            | tracks api request | kong.\<api_name>.request.count
+`request_size`             | tracks api request's body size in bytes | kong.\<api_name>.request.size
+`response_size`            | tracks api response's body size in bytes | kong.\<api_name>.response.size
+`latency`                  | tracks the time interval between the request started and response received from the upstream server | kong.\<api_name>.latency
+`status_count`             | tracks each status code returned as response | kong.\<api_name>.status.\<status>.count and kong.\<api_name>.status.\<status>.total
+`unique_users`             | tracks unique users made a request to the api | kong.\<api_name>.user.uniques
```
Comment (theetrain) [inline app/plugins/statsd.md]: > tracks unique users `who` made a request to the api

**Comment by shashiranjan84** [comment]
Blocked by https://github.com/Mashape/kong/pull/2400

**Comment by shashiranjan84** [comment]
Closing this to open a new one against `0.11.0`

**Pull request closed without merging**